[2023-11-21 06:00:21,374] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 06:00:21,414] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 06:00:21,415] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 06:00:21,416] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 06:00:21,417] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 06:00:21,448] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 06:00:21,456] {standard_task_runner.py:52} INFO - Started process 4360 to run task
[2023-11-21 06:00:21,463] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpjm98nsxj', '--error-file', '/tmp/tmpca9hj51v']
[2023-11-21 06:00:21,464] {standard_task_runner.py:80} INFO - Job 238: Subtask load_core_dim_area
[2023-11-21 06:00:21,583] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host f44f09cebae7
[2023-11-21 06:00:21,724] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 06:00:21,739] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 06:00:21,775] {dbapi.py:231} INFO - Running statement: INSERT INTO core.dim_area (area_id, name)
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy, parameters: None
[2023-11-21 06:00:21,815] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 211, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 235, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "dim_area_area_id_key"
DETAIL:  Key (area_id)=(53) already exists.

[2023-11-21 06:00:21,835] {taskinstance.py:1420} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T060021, end_date=20231121T060021
[2023-11-21 06:00:21,860] {standard_task_runner.py:97} ERROR - Failed to execute job 238 for task load_core_dim_area (duplicate key value violates unique constraint "dim_area_area_id_key"
DETAIL:  Key (area_id)=(53) already exists.
; 4360)
[2023-11-21 06:00:21,892] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-11-21 06:00:22,045] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21 08:22:37,094] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 08:22:37,111] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 08:22:37,112] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 08:22:37,112] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 08:22:37,113] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 08:22:37,136] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 08:22:37,142] {standard_task_runner.py:52} INFO - Started process 5134 to run task
[2023-11-21 08:22:37,146] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '246', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp76ihf6y4', '--error-file', '/tmp/tmpn4qhgkhd']
[2023-11-21 08:22:37,147] {standard_task_runner.py:80} INFO - Job 246: Subtask load_core_dim_area
[2023-11-21 08:22:37,235] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host f44f09cebae7
[2023-11-21 08:22:37,332] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 08:22:37,344] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 08:22:37,366] {dbapi.py:231} INFO - Running statement: INSERT INTO core.dim_area (area_id, name)
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy, parameters: None
[2023-11-21 08:22:37,377] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 211, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 235, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "dim_area_area_id_key"
DETAIL:  Key (area_id)=(53) already exists.

[2023-11-21 08:22:37,393] {taskinstance.py:1420} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T082237, end_date=20231121T082237
[2023-11-21 08:22:37,413] {standard_task_runner.py:97} ERROR - Failed to execute job 246 for task load_core_dim_area (duplicate key value violates unique constraint "dim_area_area_id_key"
DETAIL:  Key (area_id)=(53) already exists.
; 5134)
[2023-11-21 08:22:37,438] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-11-21 08:22:37,514] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21 15:09:15,531] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 15:09:15,571] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 15:09:15,572] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 15:09:15,573] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 15:09:15,575] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 15:09:15,658] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 15:09:15,671] {standard_task_runner.py:52} INFO - Started process 1180 to run task
[2023-11-21 15:09:15,686] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpionupdv3', '--error-file', '/tmp/tmpg89daox9']
[2023-11-21 15:09:15,688] {standard_task_runner.py:80} INFO - Job 254: Subtask load_core_dim_area
[2023-11-21 15:09:15,953] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host f44f09cebae7
[2023-11-21 15:09:16,206] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 15:09:16,226] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 15:09:16,266] {dbapi.py:231} INFO - Running statement: 
        MERGE INTO core.dim_area AS trg
        USING(
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy) AS src
        ON trg.area_id = src.area_id
        WHEN NOT MATCHED
        THEN INSERT(
            area_id,
            name)
        VALUES(
            src.area_id,
            src.area_name)
        WHEN MATCHED 
            AND (COALESCE(trg.name, '') <> COALESCE(src.area_name, '') 
            OR trg.area_id <> src.area_id )
        THEN UPDATE
        SET 
            area_id = src.area_id, 
            name = src.area_name, parameters: None
[2023-11-21 15:09:16,339] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T150915, end_date=20231121T150916
[2023-11-21 15:09:16,422] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-21 15:09:16,583] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-21 15:27:03,490] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 15:27:03,525] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 15:27:03,526] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 15:27:03,527] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 15:27:03,528] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 15:27:03,573] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 15:27:03,586] {standard_task_runner.py:52} INFO - Started process 287 to run task
[2023-11-21 15:27:03,594] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '271', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpr9q2yb20', '--error-file', '/tmp/tmpzzpih09b']
[2023-11-21 15:27:03,596] {standard_task_runner.py:80} INFO - Job 271: Subtask load_core_dim_area
[2023-11-21 15:27:03,763] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host b47ed23e9d21
[2023-11-21 15:27:04,078] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 15:27:04,122] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 15:27:04,185] {dbapi.py:231} INFO - Running statement: 
        MERGE INTO core.dim_area AS trg
        USING(
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy) AS src
        ON trg.area_id = src.area_id
        WHEN NOT MATCHED
        THEN INSERT(
            area_id,
            name)
        VALUES(
            src.area_id,
            src.area_name)
        WHEN MATCHED 
            AND (COALESCE(trg.name, '') <> COALESCE(src.area_name, '') 
            OR trg.area_id <> src.area_id )
        THEN UPDATE
        SET 
            area_id = src.area_id, 
            name = src.area_name, parameters: None
[2023-11-21 15:27:04,259] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T152703, end_date=20231121T152704
[2023-11-21 15:27:04,369] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-21 15:27:04,554] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-21 15:44:08,202] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 15:44:08,222] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 15:44:08,223] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 15:44:08,224] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 15:44:08,224] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 15:44:08,248] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 15:44:08,255] {standard_task_runner.py:52} INFO - Started process 1048 to run task
[2023-11-21 15:44:08,258] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp815babw2', '--error-file', '/tmp/tmp6opxd2_t']
[2023-11-21 15:44:08,259] {standard_task_runner.py:80} INFO - Job 285: Subtask load_core_dim_area
[2023-11-21 15:44:08,354] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host b47ed23e9d21
[2023-11-21 15:44:08,481] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 15:44:08,495] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 15:44:08,522] {dbapi.py:231} INFO - Running statement: 
        MERGE INTO core.dim_area AS trg
        USING(
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy) AS src
        ON trg.area_id = src.area_id
        WHEN NOT MATCHED
        THEN INSERT(
            area_id,
            name)
        VALUES(
            src.area_id,
            src.area_name)
        WHEN MATCHED 
            AND (COALESCE(trg.name, '') <> COALESCE(src.area_name, '') 
            OR trg.area_id <> src.area_id )
        THEN UPDATE
        SET 
            area_id = src.area_id, 
            name = src.area_name, parameters: None
[2023-11-21 15:44:08,560] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T154408, end_date=20231121T154408
[2023-11-21 15:44:08,592] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-21 15:44:08,685] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-21 16:03:00,617] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 16:03:00,640] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 16:03:00,641] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 16:03:00,641] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 16:03:00,642] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 16:03:00,665] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 16:03:00,673] {standard_task_runner.py:52} INFO - Started process 1914 to run task
[2023-11-21 16:03:00,677] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3rysqwtw', '--error-file', '/tmp/tmp1x_cx8v1']
[2023-11-21 16:03:00,678] {standard_task_runner.py:80} INFO - Job 303: Subtask load_core_dim_area
[2023-11-21 16:03:00,780] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host b47ed23e9d21
[2023-11-21 16:03:00,878] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 16:03:00,890] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 16:03:00,912] {dbapi.py:231} INFO - Running statement: 
        MERGE INTO core.dim_area AS trg
        USING(
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy) AS src
        ON trg.area_id = src.area_id
        WHEN NOT MATCHED
        THEN INSERT(
            area_id,
            name)
        VALUES(
            src.area_id,
            src.area_name)
        WHEN MATCHED 
            AND (COALESCE(trg.name, '') <> COALESCE(src.area_name, '') 
            OR trg.area_id <> src.area_id )
        THEN UPDATE
        SET 
            area_id = src.area_id, 
            name = src.area_name, parameters: None
[2023-11-21 16:03:00,941] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T160300, end_date=20231121T160300
[2023-11-21 16:03:00,970] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-21 16:03:01,023] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-21 16:05:01,062] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 16:05:01,083] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 16:05:01,084] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 16:05:01,084] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 16:05:01,085] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 16:05:01,107] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 16:05:01,114] {standard_task_runner.py:52} INFO - Started process 2013 to run task
[2023-11-21 16:05:01,118] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '308', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpzlfn2wf_', '--error-file', '/tmp/tmpl8_nm9kv']
[2023-11-21 16:05:01,119] {standard_task_runner.py:80} INFO - Job 308: Subtask load_core_dim_area
[2023-11-21 16:05:01,214] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host b47ed23e9d21
[2023-11-21 16:05:01,335] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 16:05:01,349] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 16:05:01,374] {dbapi.py:231} INFO - Running statement: 
        MERGE INTO core.dim_area AS trg
        USING(
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy) AS src
        ON trg.area_id = src.area_id
        WHEN NOT MATCHED
        THEN INSERT(
            area_id,
            name)
        VALUES(
            src.area_id,
            src.area_name)
        WHEN MATCHED 
            AND (COALESCE(trg.name, '') <> COALESCE(src.area_name, '') 
            OR trg.area_id <> src.area_id )
        THEN UPDATE
        SET 
            area_id = src.area_id, 
            name = src.area_name, parameters: None
[2023-11-21 16:05:01,408] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T160501, end_date=20231121T160501
[2023-11-21 16:05:01,450] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-21 16:05:01,549] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-21 16:14:21,089] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 16:14:21,106] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21 16:14:21,107] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 16:14:21,107] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-11-21 16:14:21,107] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-11-21 16:14:21,126] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load_core_dim_area> on 2023-11-20 00:00:00+00:00
[2023-11-21 16:14:21,132] {standard_task_runner.py:52} INFO - Started process 2451 to run task
[2023-11-21 16:14:21,136] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_hh', 'load_core_dim_area', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '324', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpolok4lif', '--error-file', '/tmp/tmpo0nuv3tq']
[2023-11-21 16:14:21,137] {standard_task_runner.py:80} INFO - Job 324: Subtask load_core_dim_area
[2023-11-21 16:14:21,233] {task_command.py:371} INFO - Running <TaskInstance: etl_hh.load_core_dim_area scheduled__2023-11-20T00:00:00+00:00 [running]> on host b47ed23e9d21
[2023-11-21 16:14:21,356] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_hh
AIRFLOW_CTX_TASK_ID=load_core_dim_area
AIRFLOW_CTX_EXECUTION_DATE=2023-11-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-20T00:00:00+00:00
[2023-11-21 16:14:21,372] {base.py:68} INFO - Using connection ID 'my_db_conn' for task execution.
[2023-11-21 16:14:21,397] {dbapi.py:231} INFO - Running statement: 
        MERGE INTO core.dim_area AS trg
        USING(
            SELECT DISTINCT 
                area_id,
                area_name
            FROM staging.vacancy) AS src
        ON trg.area_id = src.area_id
        WHEN NOT MATCHED
        THEN INSERT(
            area_id,
            name)
        VALUES(
            src.area_id,
            src.area_name)
        WHEN MATCHED 
            AND (COALESCE(trg.name, '') <> COALESCE(src.area_name, '') 
            OR trg.area_id <> src.area_id )
        THEN UPDATE
        SET 
            area_id = src.area_id, 
            name = src.area_name, parameters: None
[2023-11-21 16:14:21,437] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_hh, task_id=load_core_dim_area, execution_date=20231120T000000, start_date=20231121T161421, end_date=20231121T161421
[2023-11-21 16:14:21,467] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-21 16:14:21,551] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
